docusaurus/docs/how-to-guides/panel-plugins/plugin-internationalization.md

```diff
@@ -0,0 +1,264 @@
+---
+id: plugin-internationalization
+title: How to add translations to your plugin
+description: Translate your plugins
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - frontend
+  - tooling
+  - configuration
+  - translation
+  - localization
+  - internationalization
+  - webpack
+---
+
+# How to add translations to your plugin
+
+The Grafana UI is available in [several languages](https://grafana.com/docs/grafana/latest/administration/organization-preferences/#change-grafana-language). By default, plugins are available in English only and are not translated when you change your language settings. 
+
+Grafana uses the [i18next internationalization framework](https://www.i18next.com/) for translation purposes. If you want your plugin to be translatable to other languages you need to perform the changes described in this document.  
+
+## Before you begin
+
+:::info
+Available in Grafana >=v11.0.0 (US English) and Grafana >=v12.1.0 for the rest of languages.
+:::
+
+The following is recommended:
+
+* Basic knowledge of Grafana panel plugin development
+* Basic understanding of the [i18next internationalization framework](https://www.i18next.com/)
+
+## Overview of the files affected by translation
+
+If you create your plugin running the `create-plugin` scaffolding tool you'll get the following folder layout and key file:
```

Comment:
Not sure what to do here, but should we mention that this `how to example` is for a scaffold panel plugin but its the same process for data source plugins and app plugins?

Reply:
Maybe we should only list the files that are affected by translation as the heading says?

<orgName>-<pluginName>-<pluginType>
├── src
│   ├── locales
│   │   ├── en-US
│   │   │   └── <orgName>-<pluginName>-<pluginType>.json
│   │   ├── pt-BR
│   │   │   └── <orgName>-<pluginName>-<pluginType>.json
│   │   └── i18next-parser.config.js
│   ├── module.ts
│   └── plugin.json
├── docker-compose.yaml
├── web.config.ts
└── package.json